addons/l10n_pe_website_sale/static/src/js/website_sale.js

```diff
@@ -0,0 +1,90 @@
+/** @odoo-module **/
+import {WebsiteSale} from "@website_sale/js/website_sale";
+
+WebsiteSale.include({
+    events: Object.assign({}, WebsiteSale.prototype.events, {
+        "change select[name='state_id']": "_onChangeState",
+        "change select[name='city_id']": "_onChangeCity",
+    }),
+    start: function () {
+        this.selectCities = document.querySelector("select[name='city_id']");
+        this.selectDistricts = document.querySelector("select[name='l10n_pe_district']");
+        this.cityBlock = document.querySelector(".div_city");
+        this.autoFormat = document.querySelector(".checkout_autoformat");
+        this.selectState = document.querySelector("select[name='state_id']");
+        this.selectCountry = document.querySelector("select[name='country_id']");
```

Comment:
I would prefer better names - indicating these are DOM elements
maybe something like `this.cityElement` or `this.cityDivElement` or `countrySelectElement`

Reply:
Done @h4818